lgc/patch/Gfx9ConfigBuilder.cpp

```diff
@@ -98,7 +98,10 @@ void ConfigBuilder::buildPalMetadata() {
 // Builds register configuration for graphics pipeline (VS-FS), or FS-only shader compilation
 void ConfigBuilder::buildPipelineVsFsRegConfig() {
   GfxIpVersion gfxIp = m_pipelineState->getTargetInfo().getGfxIpVersion();
-  assert(gfxIp.major <= 10);
+  const bool partPipeline = !m_pipelineState->isWholePipeline() && m_pipelineState->hasShaderStage(ShaderStageFragment);
+  if (!partPipeline)
+    assert(gfxIp.major <= 10); // If not part-pipeline compilation, must <= GFX10
+  (void(partPipeline));        // Unused
```

Comment:
You could turn this into
```
assert(gfxIp.major <= 10 || partPipeline);
```
Otherwise, partPipeline isn't really unused :)

Reply:
Oh. You are right :)